sdk/typescript/package.json

```diff
@@ -28,13 +28,13 @@
     "codegen:version": "node genversion.mjs",
     "dev": "pnpm build:tsup --watch",
     "build": "node genversion.mjs && pnpm build:types && pnpm build:tsup",
-    "build:tsup": "tsup ./src/index.ts --format esm,cjs --sourcemap",
+    "build:tsup": "tsup ./src/index.ts --format esm,cjs --sourcemap --env.SUI_TEST false",
     "build:types": "tsc --build",
     "doc": "typedoc",
     "vitest": "vitest",
     "test": "pnpm test:typecheck && pnpm test:unit",
     "test:typecheck": "tsc -p ./test",
-    "test:unit": "vitest run unit __tests__",
+    "test:unit": "SUI_TEST=true vitest run unit __tests__",
```

Comment:
Why not NODE_ENV=test?

Reply:
I'll update to that. I had initially been thinking that the variable wouldn't be replaced, so I didn't want to use something customers might be setting, but since tsup is replacing this in the output, that should be safe